src/openassetio-python/cmodule/src/BatchElementErrorBinding.cpp

```diff
@@ -24,6 +25,8 @@ void registerBatchElementError(py::module &mod) {
   batchElementError
       .def(py::init<BatchElementError::ErrorCode, openassetio::Str>(), py::arg("code"),
            py::arg("message"))
+      .def(py::self == py::self)
+      .def(py::self != py::self)
```

Comment:
[consider] I believe (though not 100% sure) that `py::self != py::self` is unnecessary if `py::self == py::self` is defined.